spec/std/enum_spec.cr

```diff
@@ -37,6 +37,17 @@ describe Enum do
     end
   end
 
+  it "creates an enum instance from a (macro) symbol (#8573)" do
```

Comment:
```suggestion
  it "creates an enum instance from a symbol (#8573)" do
```

Reply:
Well I'd like to keep the wording and test the original usecase I had in mind (creating an enum instance from a macro symbol)

Reply:
IMO real usecase here is initializing enums from symbols, doing this from within the macro-generated code is just a technical detail.

thus, you could also remove the `{% begin ... end %}` macro from the specs.

Reply:
That's fair, I'll make the edits when I can